### PR TITLE
Update CVE-2018-6659 #2.json

### PR DESCRIPTION
**Pull request by htoomey**
Fixed version in JSON

### CODE REVIEW

File: 2018/6xxx/CVE-2018-6659.json [hunk 1]
```diff
@@ -1,18 +1,99 @@
...
+      }
+   },
+   "data_format": "MITRE",
+   "data_type": "CVE",
+   "data_version": "4.0",
+   "description": {
+      "description_data": [
+         {
+            "lang": "eng",
+            "value": "Reflected Cross-Site Scripting vulnerability in McAfee ePolicy Orchestrator (ePO) 5.3.2, 5.3.1, 5.3.0 and 5.9.0 allows local users to exploit an XSS issue via not sanitizing the user input."
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6659.json]: Shouldn't that say "allows **remote** authenticated users"? 

George

Comment (htoomey) [reply 2018/6xxx/CVE-2018-6659.json]: Yes. it should.  Good catch.  Go ahead and make the change.  I have updated our local database and we will update the security bulletin.

**Comment by cve-team** [comment]
I manually changed the description, then populated the entry in the CVE List. It should be published on https://cve.mitre.org within the next few hours.

George

**Comment by htoomey** [comment]
Thanks George!

**Pull request merged**